tools/windows.py

```diff
@@ -29,6 +30,9 @@ def generate(env):
             env.Append(CCFLAGS=["/Z7", "/Od", "/EHsc", "/D_DEBUG", "/MDd"])
         elif env["target"] == "release":
             env.Append(CCFLAGS=["/O2", "/EHsc", "/DNDEBUG", "/MD"])
+        if env["use_clang_cl"]:
+            env.Replace(CC="clang-cl")
+            env.Replace(CXX="clang-cl")
```

Comment:
```suggestion
            env["CC"] ="clang-cl"
            env["CXX"] ="clang-cl"
```

to match the style used in other places